geometry_msgs/Pose[] poses
float64[] target_coverages
---
int32[] ids
